packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.stories.tsx

```diff
@@ -439,3 +439,20 @@ export const TableWithJSONColumns: Story = {
     shouldDeepLink: false,
   },
 }
+
+export const MaterializedViewOfFiles: Story = {
```

Comment:
Example story demonstrating the access column and direct download column on a Materialized View